learning_wavelets/evaluation_scripts/exact_recon_unet_evaluate.py

```diff
@@ -0,0 +1,56 @@
+import tensorflow as tf
+from tqdm import tqdm
+
+from learning_wavelets.config import CHECKPOINTS_DIR
+from learning_wavelets.data.datasets import im_dataset_bsd68
+from learning_wavelets.evaluate import METRIC_FUNCS, Metrics 
+from learning_wavelets.models.exact_recon_unet import ExactReconUnet
+
+
+
+def evaluate_unet(
+        noise_std_test=30,
```

Comment:
There is going to be a problem with this default value, because it's not iterable.

I think the best way to do this is to handle the case where `noise_std_test` is not an iterable (i.e. a float or an integer most likely), and make sure to transform it into a list.